docs/user/attestation-provider.md

```diff
@@ -0,0 +1,113 @@
+# Becoming an Attestation Provider
+
+This guide is a walk through to become an [attestation provider](#attestation-provider).
+There are two situations in which it makes sense to do so:
+
+* Running local attestation providers, without any capital requirement, to act as a safeguard against malicious forks ;
+* Running an incentivized attestation provider to reap rewards, which contributes to the network and helps improve the decentralization of the network and provides attestations for the [state connector](#state-connector).
+
+??? note "Rewards"
+
+    Currently, the process of rewarding public attestation providers is still being discussed.
+
+## Prerequisites
+
+Before proceeding with the [installation](#installation), it is essential to follow every prerequisite.
+
+### System Requirements
+
+The following specifications were observed to be able to run all nodes on a single machine.
```

Comment:
I'm not sure it's clear what nodes are you talking about here.

Reply:
Run nodes for all blockchains, will clarify, thanks!

Reply:
Fixed in [`a45798b` (#14)](https://github.com/flare-foundation/docs/pull/14/commits/a45798b04b3781889e74327f15aec12634ecb536)